src/client/search/SearchActions.js

```diff
@@ -22,3 +25,24 @@ const addQueryParamTypes = (searchQuery, query, paramTypes) => {
     paramTypes,
   };
 };
+
+export const INVALIDATE_FEED_RESULTS = 'INVALIDATE_FEED_RESULTS';
```

Comment:
http://redux.js.org/docs/advanced/AsyncActions.html

fetching and invalidating are meant to represent two different actions, invalidation says "hey these results may no longer be relevant, let's maybe put up a notice or a spinner" - just basing it on the tutorial